### PR TITLE
Alter storybook config to load more static files

### PR DESCRIPTION
**Pull request by mrblippy**
##### Checklist

- [x] the [contributor license agreement][] is signed
- [x] commit message follows [commit guidelines][]
- [x] tests are included

##### Description of change

#1306 
I followed the instructions here https://storybook.js.org/docs/react/configure/images-and-assets#serving-static-files-via-storybook to load static images that are used in stories. Not 100% sure if this is the correct folder structure to use.

This will allow the SearchResult related stories to have an actual thumbnail, served up locally

Also included in this PR is small change to an action inside SearchResultList.stores, so it has a more descriptive name 

[contributor license agreement]: https://www.apereo.org/node/676
[commit guidelines]: https://chris.beams.io/posts/git-commit/

### CODE REVIEW

File: Source/Plugins/Core/com.equella.core/js/__stories__/search/SearchResultList.stories.tsx [hunk 1]
```diff
@@ -56,7 +56,7 @@ EmptyResultListComponent.args = {
     count: emptySearch.available,
   },
   orderSelectProps: {
-    onChange: action("onChange called"),
+    onChange: action("orderSelect onChange called"),
```
Comment (ChristianMurphy) [inline Source/Plugins/Core/com.equella.core/js/__stories__/search/SearchResultList.stories.tsx:59]: Is this change intended?

**Review by ChristianMurphy — APPROVED** [review]
looks good, thanks @mrblippy!

File: Source/Plugins/Core/com.equella.core/js/__stories__/search/SearchResultList.stories.tsx (hunk 1 above)
Comment (ChristianMurphy) [reply Source/Plugins/Core/com.equella.core/js/__stories__/search/SearchResultList.stories.tsx:59]: ah

> Also included in this PR is small change to an action inside SearchResultList.stores, so it has a more descriptive name

I see, it appears to be the only on change :man_shrugging: 
But it also doesn't hurt.

**Review by edalex-ian — APPROVED** [review]
Good work ~Larry~ @mrblippy :wink: .

**Review by PenghaiZhang — APPROVED** [review]
Looks good to me!
